src/components/picker.js

```diff
@@ -133,15 +138,17 @@ export default class Picker extends React.Component {
   }
 
   handleEmojiOver(emoji) {
-    var { preview } = this.refs
+    var { preview,search } = this.refs
     preview.setState({ emoji: emoji })
+    search.setState({placeholder : emoji.colons })
```

Comment:
now onMouseover emoji  the  colon will be displayed as placeholder on searchbar.